tests/robo/jenkins-robo-run.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+input=$(gsutil ls -l gs://vic-engine-builds/vic_* | grep -v TOTAL | sort -k2 -r | head -n1 | xargs | cut -d ' ' -f 3 | cut -d '/' -f 4)
+buildNumber=${input:4}
+
+n=0
+   until [ $n -ge 5 ]
+   do
+      echo "Retry.. $n"
+      echo "Downloading gcp file $input"
+      wget https://storage.googleapis.com/vic-engine-builds/$input
+      if [ -f "$input" ]
+      then
+      echo "File found.."
+      break
+      else
+      echo "File NOT found"
+      fi
+      n=$[$n+1]
+      sleep 15
+   done
+
+n=0
+   until [ $n -ge 5 ]
+   do
+      mkdir bin
+      echo "Extracting .tar.gz"
+      tar xvzf $input -C bin/ --strip 1
+      if [ -f "bin/vic-machine-linux" ]
+      then
+      echo "tar extraction complete.."
+      canContinue="Yes"
+      break
+      else
+      echo "tar extraction failed"
+      canContinue="No"
+      rm -rf bin
+      fi
+      n=$[$n+1]
+      sleep 15
+   done
+
+if [[ $canContinue = "No" ]]; then
+    echo "Tarball extraction failed..quitting the run"
+    break
+else
+    echo "Tarball extraction passed, Running nightlies test.."
+fi
+
+pybot --removekeywords TAG:secret -d robo tests/manual-test-cases/Group19-DRS-Disabled > robo/stdout.txt
+
+cat robo/stdout.txt | grep '::' | grep -E 'PASS|FAIL' > console.log
```

Comment:
Use the latest grep from vic-product (change initial grep to grep -E '::|\.\.\.'):
https://github.com/vmware/vic-product/blob/master/tests/vic-product-nightly.sh#L52